### PR TITLE
feat: add cookieConsent

### LINKED ISSUE #59: privacy and cookie policy text

### PR DESCRIPTION
**Pull request by TheJoin95**
Closes #59 

- add react-cookie-consent package
- add generic cookie policy, monolanguage
- add link in cookie policy banner

image

### CODE REVIEW

**Comment by nicpuppa** [comment]
@TheJoin95 do we need any cookies policy page or similar ?

**Comment by TheJoin95** [comment]
@nicpuppa added

File: pages/cookie.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            <h4>What are cookies?</h4>
+            <p>
+              Cookies are simple text files that are stored on your computer or mobile device by a website&apos;s server. Each cookie is unique to your web browser.
+              It will contain some anonymous information such as a unique identifier, website&apos;s domain name, and some digits and numbers.
+            </p>
+            <h4>What types of cookies do we use?</h4>
+            <h5>Necessary cookies</h5>
+            <p>
+              Necessary cookies allow us to offer you the best possible experience when accessing and navigating through our website and using its features.
+              For example, these cookies let us recognize that you have created an account and have logged into that account.
```
Comment (Wabri) [inline pages/cookie.tsx:38]: We did not have a account feature!

File: pages/cookie.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            </p>
+            <h5>Functionality cookies</h5>
+            <p>
+              Functionality cookies let us operate the site in accordance with the choices you make.
+              For example, we will recognize your username and remember how you customized the site during future visits.
```
Comment (Wabri) [inline pages/cookie.tsx:43]: We did not have a account feature!

**Review by nicpuppa — APPROVED** [review]

**Review by Wabri — CHANGES_REQUESTED** [review]

**Review by 404answernotfound — CHANGES_REQUESTED** [review]
Maybe we should change explainatory examples

File: pages/cookie.tsx (hunk 1 above)
Comment (TheJoin95) [reply pages/cookie.tsx]: Updated

File: pages/cookie.tsx (hunk 2 above)
Comment (TheJoin95) [reply pages/cookie.tsx]: Updated

**Comment by TheJoin95** [comment]
> 

Updated with example in context

**Review by Wabri — APPROVED** [review]

**Pull request merged**